config/application.rb

```diff
@@ -146,6 +146,9 @@ class Application < Rails::Application
         child.helper helpers
       end
     end
+
+    # Should ANSI color codes be used when logging information
+    config.colorize_logging = SETTINGS[:colorize_logging]
```

Comment:
@eLobato putting this here would cause it to be defaulted to off in all environments.
Is this what you meant or should I create a setting for each env?